.github/workflows/on_push.yml

```diff
@@ -105,3 +105,23 @@ jobs:
         --container api-container \
         -- /opt/app/bin/liquid_voting \
         eval "LiquidVoting.Release.migrate"
+        eval "LiquidVoting.Release.smoke_test_teardown"
```

Comment:
should be run after smoke tests, else the data from the latter will stick around on the db until another deployment is done. Also not sure this second eval actually executes. 

It's ok to have a similar command to the deploy one at the end of the smoke test. it's not really duplicate

Reply:
Thanks for moving the PR. I'll go back to draft mode + experimenting in on_pull_request.yml :)

About deleting after the tests. I thought about that, and can't see any advantage. Surely, you (or someone) will need to run the Realease.teardown_test_data function manually for any further tests following deploy? (deleting after only gives you credit for one more)

Reply:
... except there is no draft mode available now.

Reply:
it's already set to draft ;)

Reply:
> Thanks for moving the PR. I'll go back to draft mode + experimenting in on_pull_request.yml :)
> 
> About deleting after the tests. I thought about that, and can't see any advantage. Surely, you (or someone) will need to run the Realease.teardown_test_data function manually for any further tests following deploy? (deleting after only gives you credit for one more)

it's good practice to teardown and cleanup after a CI job does its work. and they're separate concerns - the teardown doesn't relate to the deploy at all.

Reply:
and this will always ensure that there's no leftover data there when doing manual tests in between CI runs

Reply:
I am happy to add a teardown after, but any subsequent test run will leave data that will require teardown before another test run.

Reply:
sure, any test run, automatic or manual, needs a teardown afterwards if there's state leftover from it.

Reply:
point is, there should always be a clean slate before running them